src/main/java/core/basesyntax/main/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.main;
+
+import core.basesyntax.state.Figure;
+import core.basesyntax.state.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] randomFigureArray = new Figure[(int)((Math.random() * 5) + 1)];
```

Comment:
What is 5? Magic number? Let's create a constant for it

Reply:
done